src/js/theme.js

```diff
@@ -516,10 +516,11 @@ class Theme {
     initMermaid() {
         const $mermaidElements = document.getElementsByClassName('mermaid');
         if ($mermaidElements.length) {
-            mermaid.initialize({startOnLoad: false, theme: 'null'});
+            mermaid.initialize({startOnLoad: false, theme: 'default'});
```

Comment:
`theme: null` does work in older version of mermaid but does not work in `^8.9.*` and set theme to `null` or `"null"` will cause different problems. This might be the issue with mermaid itself.

According to the docs, `null` should be a valid option: https://mermaid-js.github.io/mermaid/#/Setup?id=theme

Change theme to `default` or anything other than `null` will cause add inline `<style>` to the diagrams and overwrite the custom CSS, which result in a bad experience in dark mode.